structuralcodes/codes/mc2010/_concrete_material_properties.py

```diff
@@ -1,6 +1,9 @@
 """A collection of material properties for concrete."""
 
+from __future__ import annotations
```

Comment:
This is a very good question! You can read a good summary [here](https://blog.derlin.ch/python-type-hints-and-future-annotations). One use of this import is that it allows us to use "more modern" type hints, e.g. the pipe operator `|` as a replacement for `Union`, in 3.9 and below. I guess that was the intention of the import at the time this particular code was written, but right now, I cannot see why it is needed. I will have a closer look and remove if needed.

Reply:
Turns out we still had a couple of `tuple` instead of `t.Tuple`, and a couple of annotate returns of methods of classes where the methods returned instances of the classes themselves. I have removed all imports from `__future__`, replaced the `tuple` with `t.Tuple`, and replaced some return annotations with strings e.g. `'Geometry'` instead of `Geometry`.

Reply:
After merging dev, and re-building the docs, I recalled why the `from __future__ import annotations` was added. It was to prevent all annotations of `ArrayLike` to be transformed to `Buffer | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes]` in the automatically created API docs.

I am adding again the import.

Nevertheless, this helped ruling out a couple of code inconsistencies 😃

Reply:
Perfect, sorry If I slowed down the process with my question.